lib/rucio/core/config.py

```diff
@@ -116,7 +117,7 @@ def get(section, option, session=None):
     if tmp is not None:
         return __convert_type(tmp[0])
     else:
-        raise ConfigNotFound()
+        return default
```

Comment:
I think this should be handled differently. If `default=None` and nothing can be found the exception should still be raised. Only otherwise we raise the default.
This should be fine, since `None` is not an eligible default value anyway (since the value comes from the db)

Reply:
ah yes make sense